spring-cloud-stream-samples/pom.xml

```diff
@@ -21,6 +21,7 @@
 		<module>tap</module>
 		<module>double</module>
 		<module>extended</module>
+		<module>rxjava</module>
```

Comment:
Let's move this to Spring Cloud Stream Modules and follow the same strategy as for XD: provide an OOTB library for building RxJava modules based on `@EnableModule` support. On the long run we'll want to have this provided via `@Input Observable` etc, but that's a lofty goal for now.